_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,78 @@
+
+You can query your Connector's Address by proceeding as documented in the [Get currently used Identity]({% link _docs_use-cases/use-case-transport-get-currently-used-identity.md %}) use case.
+{: .notice--info}
+
+## Process
+
+As you can see from the diagram below, after you have entered the input values to create an IdentityAttribute for your own Connector, a few checks are performed to verify the correctness of your input. If the input is not entered correctly, an [error message]({% link _docs_integrate/error-codes.md %}) is sent in response. Otherwise, a check is performed whether the input values for the properties of the specified IdentityAttribute Value meet the validation criteria documented on the [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) page. Assuming a successful validation, the IdentityAttribute to be created is saved as a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). If it is a simple IdentityAttribute, a success response is sent directly. In the case of a complex IdentityAttribute, on the other hand, another LocalAttribute is created beforehand for each of its appropriate components. These LocalAttributes of the components contain the `id` of the LocalAttribute belonging to the complex IdentityAttribute within their `parentId` property.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/310cea0e-6f6f-4ee0-9efd-55e180ec5dda" id="WT4OFNWd3bcS"></iframe></div>
+
+### Example 1: Create an own simple IdentityAttribute
+
+An example of a simple IdentityAttribute is one of type [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname). If you want to create it without specifying optional parameters, you must use the following content:
+
+```jsonc
+{
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<Address of your Connector>",
+    "value": {
+      "@type": "DisplayName",
+      "value": "<display name of your Connector>"
+    }
+  }
+}
+```
+
+Assuming that the input value for the Connector's display name specified in the `value.value` property meets the [validation criterion]({% link _docs_integrate/attribute-values.md %}#displayname), which means that the entered name is not more than 100 characters long, the IdentityAttribute is saved as a LocalAttribute and a success response is sent.
+
+### Example 2: Create an own complex IdentityAttribute
+
+An example of a complex IdentityAttribute is one of type [BirthDate]({% link _docs_integrate/attribute-values.md %}#birthdate). If you want to create it without specifying optional parameters, you must use the following content:
+
+```jsonc
+{
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<Address of your Connector>",
+    "value": {
+      "@type": "BirthDate",
+      "day": <day of birth>,
```

Comment:
In another PR you wrote "" around the values of the child attributes. I commented on it and you argued that they can be specified like this. I'm not sure anymore, what was the outcome of that conversation, but perhaps check, that we have it consistently in the scenarios.

Reply:
Ok, I just saw that this is consistent with the merged scenarios.